src/main/scala/beam/router/Modes.scala

```diff
@@ -245,3 +247,69 @@ object Modes {
   }
 
 }
+
+object TourModes {
+  import beam.router.Modes.BeamMode
+  import beam.router.Modes.BeamMode._
+
+  sealed abstract class BeamTourMode(
+    val value: String,
+    val vehicleCategory: VehicleCategory,
+    val allowedBeamModes: Seq[BeamMode],
+    val allowedBeamModesForFirstAndLastLeg: Seq[BeamMode]
+  ) extends StringEnumEntry {
+
+    import BeamTourMode._
+
+    def isVehicleBased: Boolean = this match {
+      case WALK_BASED => false
+      case _          => true
+    }
+  }
+
+  object BeamTourMode extends StringEnum[BeamTourMode] with StringCirceEnum[BeamTourMode] {
+
+    override val values: immutable.IndexedSeq[BeamTourMode] = findValues
+
+    val enabledModes: Map[BeamMode, Seq[BeamMode]] =
+      Map[BeamMode, Seq[BeamMode]](CAR -> Seq(DRIVE_TRANSIT), BIKE -> Seq(BIKE_TRANSIT))
+
+    // TODO: Also allow use of shared bikes/cars in walk based tours
+    case object WALK_BASED
+        extends BeamTourMode(
+          "walk_based",
+          Body,
+          Seq[BeamMode](
+            WALK,
+            WALK_TRANSIT,
+            RIDE_HAIL,
+            RIDE_HAIL_POOLED,
+            RIDE_HAIL_TRANSIT,
+            HOV2_TELEPORTATION,
+            HOV3_TELEPORTATION
+          ),
+          Seq[BeamMode](
+            WALK,
+            WALK_TRANSIT,
+            RIDE_HAIL,
+            RIDE_HAIL_POOLED,
+            RIDE_HAIL_TRANSIT,
+            DRIVE_TRANSIT,
+            BIKE_TRANSIT,
+            HOV2_TELEPORTATION,
+            HOV3_TELEPORTATION
+          )
+        )
+
+    case object CAR_BASED
+        extends BeamTourMode(
+          "car_based",
+          Car,
+          Seq[BeamMode](CAR, CAR_HOV2, CAR_HOV3),
```

Comment:
How about drive_transit? This might result drive_transit be categorized into walk_based tour
![image](https://user-images.githubusercontent.com/58761221/172474284-50e033e6-894e-4fde-9251-dd7322f8c33a.png)